src/components/DownloadComponents/DownloadConfirmation/DownloadConfirmation.tsx

```diff
@@ -5,79 +5,103 @@ import { Heading, P } from "@/components/Typography";
 import Svg from "@/components/Svg";
 import Box from "@/components/Box";
 import ButtonAsLink from "@/components/Button/ButtonAsLink";
+import NextLessonContainer from "@/components/DownloadComponents/NextLessonContainer";
+import { NextLesson } from "@/node-lib/curriculum-api-2023/queries/lessonDownloads/lessonDownloads.schema";
 
 type DownloadConfirmationProps = {
   lessonSlug: string;
   programmeSlug: string | null;
   unitSlug: string | null;
+  isCanonical: boolean;
+  unitTitle?: string | null;
+  nextLessons?: NextLesson[];
```

Comment:
I'm not sure this is optional as the query method returns an empty array if there are no next lessons - possibly the canonical page is different?

Reply:
confirmed that canonical downloads page does not include next lessons